slack-go.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+define("GOSERVER", "http://go.brandwatch.com/");
+define("GOSERVERSHORT", "http://go/");
+define("GOSERVERDISPLAY", "go");
+define("BOTNAME", "go");
+define("INCOMINGWEBHOOK", "https://hooks.slack.com/services/xxx"); //Use your own incoming webhook URL here
+
+function get_long_url( $slug ) 
+{ 
+    $options = array( 
+        CURLOPT_RETURNTRANSFER => true,     // return web page 
+        CURLOPT_HEADER         => false,    // return headers 
+    ); 
+ 
+    $ch = curl_init( GOSERVER . '_api/go-link.json' ); 
+    curl_setopt_array( $ch, $options ); 
+    $content = curl_exec( $ch ); 
+ 
+    if(curl_errno($ch)) {
+        die ("An error occured (is the Go server up?)"); //problem with the go server
+    }
+
+    curl_close( $ch ); 
+
+    $links = json_decode($content, true);
+
+    foreach ($links as $link) {
+    	if ($link['shortUri'] == $slug) {
+            return $link['longUri'];
+    	}
+    }
+    die ("No link found for " . $slug);
+}
+
```

Comment:
I gather that this is where the integration actually starts doing things, but I can't quite follow what and how. Would it be possible to break things up into more functions with one main() for instance? E.g. perhaps the second curl could be put into its own function.